spec/Section 7 -- Response.md

```diff
@@ -10,9 +10,12 @@ the case that any _field error_ was raised on a field and was replaced with
 
 ## Response Format
 
-A response to a GraphQL request must be a map or a response stream of maps. When
-the response of the GraphQL operation is a response stream, the first value will
-be an initial payload, followed by one or more subsequent payloads.
+A response to a GraphQL request must be a map or a stream of incrementally
+delivered results. The response will be a stream of incrementally delivered
+results when the GraphQL service has acted on `@defer` or `@stream` directives
```

Comment:
Nit: is inlining a form of acting….?

Maybe “may be…when the graphql service has encountered” although that’s certainly vaguer